src/state/reducers/test.js

```diff
@@ -1,7 +1,54 @@
-import reducers from './index';
+import { LOGIN_ERROR } from '../../constants/errors';
+import { REQUEST_LOGIN_ACTION_SUCCESS, REQUEST_LOGIN_ACTION_FAILURE } from '../actions/actionTypes';
 
-describe('Reducers', () => {
-  test('Middlewares are added in dev', () => {
-    expect(reducers()).toBeDefined();
+import rootReducer from './index';
+import sessionReducer from './session';
+
+describe('STATE - Reducers', () => {
+  const initialSessionState = {
+    user: null,
+    errors: {},
+  };
+
+  test('reducers are combined successfully', () => {
+    // WIP: improve this test
+    expect(rootReducer).toBeDefined();
+  });
+
+  test('sessionReducer - returns initial state', () => {
+    const mockedAction = { type: 'some type of action' };
+    expect(sessionReducer(undefined, mockedAction)).toEqual(initialSessionState);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected use of undefined.](https://app.codacy.com/manual/iagoregiani/verificador-associacao/pullRequest?prid=5574829)